lib/OpenQA/Schema/Result/Workers.pm

```diff
@@ -215,10 +215,10 @@ sub send_command {
     my $client = OpenQA::WebSockets::Client->singleton;
     try { $client->send_msg($self->id, $args{command}, $args{job_id}) }
     catch ($e) {
-        log_error(
+        log_info(
```

Comment:
Tests are failing. `log_info` needs to be imported in line 11, I guess that's the reason.